### PR TITLE
make express-recaptcha - Typescript compliant

### PR DESCRIPTION
**Pull request by pdupavillon**
As described in issue #13 the actual express-recaptcha,
returns an instance instead of a class, which is hard to define in TS

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (SimonSchick) [inline .babelrc]: I'd argue that this is not really needed anymore?
You can pretty safely use es6 with node and restrict support to to NodeJS 6+ see https://github.com/nodejs/Release

File: src/express-recaptcha.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import Https from 'https'
+export default class Recaptcha{
```
Comment (SimonSchick) [inline src/express-recaptcha.js]: I know it's tempting but `default` doesn't play well with `require`, it it's somewhat annoying to use.

```const Recaptcha = require('recaptcha').default;```

Our team generally prefers to just `export`, plays nice with destructuring too!

```const { Recaptcha } = require('recaptcha')```

File: .babelrc (hunk 1 above)
Comment (SimonSchick) [reply .babelrc]: Alternatively you could give TypeScript a shot, it does everything babel does, and more!

**Comment by pdupavillon** [comment]
@SimonSchick,
Thanks a lot for your feedbacks !

**Comment by SimonSchick** [comment]
I think you misunderstood my comment about `export`, I didn't mean you should use `module.exports` but write `export class Recaptcha`

**Comment by pdupavillon** [comment]
I think I misunderstood the concept behind this :
Why should I use `exports.Recaptcha = Rechaptcha` instead of `exports Recaptcha` ?
and why `exports` instead of `module.exports` ? (aren't they almost equivalent ?)
Thanks for your explanations

**Pull request merged**